agency-api-spec.md

```diff
@@ -157,6 +157,118 @@ JSON payload that contains the form fields.
 **Required:** | no
 **Example:** | `[{"filename": "letter.pdf", "content_type": "application/pdf", "filesize": 27556, "filedata": "YSBiYXNlNjQgZW5jb2RlZCBmaWxlCg=="}]`
 
+**Field:** | `agency_form_fields`
+:--- |:---
+**Type:** | object
+**Description:** | Agency specific request form fields as specified in your [agency's metadata file](https://github.com/18F/foia-recommendations/blob/master/schemas.md#agency-metadata-file).
+**Required:** | if applicable
+**Example:** | `{"form_460A_case_number": "3347", "requester_type": "Journalist"}`
+
+
+#### Agency form fields
+
+Your agency might have additional fields specified in your [agency metadata file](https://github.com/18F/foia-recommendations/blob/master/schemas.md#agency-metadata-file).
+These additional fields are unique to your agency and are captured separately in
+the `agency_form_fields` object.
+
+The specific fields in `agency_form_fields` will be defined by your metadata
+file which includes both required and optional form fields. Any fields in
+`required_form_fields` will be considered required. Any fields in
+`additional_form_fields` will be considered optional. The portal will validate
+that any required fields are present.
```

Comment:
Do you think it's worth clarifying what this means from the agency perspective and/or being clearer that this validation isn't an implementation detail related to the process exposing this API?

Maybe rewording this last sentence to something like _DOJ's central FOIA portal will ensure that required fields are present before POSTing a request to your endpoint._